config/controller/front.config.php

```diff
@@ -10,6 +10,10 @@
             'view' => 'noviusos_form::front/form',
             // The view params
             'view_params' => array(
+                // The JS params injected before other scripts in "noviusos_form_params" JS global variable
+                'js_params_injected_inline' => array(
```

Comment:
I would have set js_params_injected_inline at the root of the config array because it does not dependent on the layout.